vendor/usercss-meta/README.md

```diff
@@ -0,0 +1,5 @@
+## usercss-meta v0.8.1
```

Comment:
Update to v0.8.2?

Reply:
I think it is better to create another PR to update libraries after this branch is merged.